src/Controls/tests/TestCases/MauiProgram.cs

```diff
@@ -8,18 +9,31 @@ namespace Maui.Controls.Sample
 {
 	public static class MauiProgram
 	{
-		public static MauiApp CreateMauiApp() =>
-			MauiApp
-				.CreateBuilder()
-	#if IOS || ANDROID
-				.UseMauiMaps()
-	#endif
-				.UseMauiApp<App>()
+		public static MauiApp CreateMauiApp()
+		{
+			var appBuilder = MauiApp.CreateBuilder();
+
+#if IOS || ANDROID
+			appBuilder.UseMauiMaps();
+#endif
+			appBuilder.UseMauiApp<App>()
 				.ConfigureFonts(fonts =>
 				{
 					fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
-				})
-				.Build();
+				});
+
+			Microsoft.Maui.Handlers.EntryHandler.Mapper.AppendToMapping(nameof(DecimalEntry), (handler, view) =>
```

Comment:
Maybe this should be moved near the test code so it is easier to maintain control once we start adding more things.

Reply:
Moved to the test class.